stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/preview/ChatPreviewViewModel.kt

```diff
@@ -0,0 +1,81 @@
+package io.getstream.chat.ui.sample.feature.chat.preview
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.viewModelScope
+import io.getstream.chat.android.client.ChatClient
+import io.getstream.chat.android.client.call.await
+import io.getstream.chat.android.livedata.ChatDomain
+import io.getstream.chat.android.livedata.utils.Event
+import io.getstream.chat.ui.sample.common.CHANNEL_ARG_DRAFT
+import kotlinx.coroutines.launch
+
+class ChatPreviewViewModel(
+    private val memberId: String,
+    private val chatDomain: ChatDomain = ChatDomain.instance(),
+    private val chatClient: ChatClient = ChatClient.instance(),
+) : ViewModel() {
+
+    private var cid: String? = null
+    private val _state: MutableLiveData<State> = MutableLiveData()
+    private val _events: MutableLiveData<Event<UiEvent>> = MutableLiveData()
+    val state: LiveData<State> = _state
+    val events = _events
+
+    init {
+        _state.value = State(cid = null)
+        viewModelScope.launch {
+            val result = ChatClient.instance()
```

Comment:
Why do you use ChatClient directly? Why not `UseCaseHelper::createChannel`?

Reply:
`CreateChannel` use case doesn't support creating `distinct` channels (based on members and not on id and type). I'll task for that